engine/flink/components/base/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/DailyWindowsOffsetDependingOnTimezone.scala

```diff
@@ -0,0 +1,23 @@
+package pl.touk.nussknacker.engine.flink.util.transformer.aggregate
+
+import java.time.ZoneId
+import java.util.TimeZone
+import java.util.concurrent.TimeUnit
+import scala.concurrent.duration.Duration
+
+object DailyWindowsOffsetDependingOnTimezone {
+
+  def offset(windowDuration: Duration, dailyWindowsAlignZoneId: ZoneId): Option[Duration] =
+    if (DailyWindowDetector.isDailyWindow(windowDuration)) {
+      val offsetInMillis = TimeZone.getTimeZone(dailyWindowsAlignZoneId).getRawOffset * -1
+      Some(Duration.apply(offsetInMillis, TimeUnit.MILLISECONDS))
+    } else {
+      None
+    }
+
+}
+
+object DailyWindowDetector {
```

Comment:
It's in the same file just for convnience. This class has single method which is used only in `DailyWindowsOffsetDependingOnTimezone` - which is in this file. 

I can extract this class to another file or move this single method to `DailyWindowsOffsetDependingOnTimezone` and make it private (but this would also mean deleting unit tests of this method) - which option do you prefer? I think 1st one is better